File: app/src/main/java/com/example/cesartorres/testmvpbus/presentation/ui/activity/ContactListActivity.java
package com.example.cesartorres.testmvpbus.presentation.ui.activity;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.example.cesartorres.testmvpbus.R;
import com.example.cesartorres.testmvpbus.presentation.ui.fragment.UserDetailFragment;
import com.example.cesartorres.testmvpbus.presentation.ui.fragment.UserListFragment;

/**
 * Created by cesar.torres on 3/9/2018.
 */

public class ContactListActivity extends AppCompatActivity{

    public static Intent newIntent(Context context){
        Intent intent = new Intent(context,ContactListActivity.class);
        return intent;
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test);
        initFragments();
    }

    private void initFragments() {
        UserListFragment userListFragment = UserListFragment.newInstance();
        getSupportFragmentManager().beginTransaction().add(R.id.fl_left_layout, userListFragment).addToBackStack(null).commit();

        UserDetailFragment userDetailFragment = UserDetailFragment.newInstance();
        getSupportFragmentManager().beginTransaction().add(R.id.fl_right_layout, userDetailFragment).addToBackStack(null).commit();
    }

    @Override
    public void onBackPressed() {
        finish();
    }
}

File: app/src/main/java/com/example/cesartorres/testmvpbus/presentation/ui/activity/ContactListActivity.kt
package com.example.cesartorres.testmvpbus.presentation.ui.activity

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.example.cesartorres.testmvpbus.R

/**
 * Created by cesar.torres on 3/9/2018.
 */
class ContactListActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)
    }

    companion object {
        fun newIntent(context: Context): Intent{
            val intent = Intent(context, ContactListActivity :: class.java)
            return intent
        }
    }
}